cmd/create/operatorroles/cmd.go

```diff
@@ -228,6 +234,18 @@ func run(cmd *cobra.Command, argv []string) {
 			ocm.Response:  ocm.Success,
 		})
 	case aws.ModeManual:
+		err = aws.GeneratePolicyFiles(r.Reporter, env, false, true, policies, credRequests)
+		if err != nil {
+			r.Reporter.Errorf("There was an error generating the policy files: %s", err)
+			r.OCMClient.LogEvent("ROSACreateOperatorRolesModeManual", map[string]string{
+				ocm.Response: ocm.Failure,
+			})
+			os.Exit(1)
+		}
+		if r.Reporter.IsTerminal() {
+			r.Reporter.Infof("All policy files saved to the current directory")
+			r.Reporter.Infof("Run the following commands to create the operator roles and policies:\n")
```

Comment:
This message is already shown below, maybe update that one instead (and remove this one, since it displays before the command is built)

Reply:
Done.